Gemfile

```diff
@@ -22,7 +22,7 @@ gem "i18n-country-translations"
 gem "rails-i18n"
 gem "translation"
 group :development, :test do
-  gem "i18n-tasks"
+  # gem "i18n-tasks"
```

Comment:
Something needs to be done about `rails-i18n` and `i18n-tasks`
```
Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    rails (= 5.2.3) was resolved to 5.2.3, which depends on
      railties (= 5.2.3)

    i18n-tasks was resolved to 0.9.29, which depends on
      rails-i18n was resolved to 4.0.9, which depends on
        railties (~> 4.0)
```

Reply:
`bundle update rails-i18n` after updating rails should do it. Barring that, setting the version explicitly:

> gem 'rails-i18n', '~> 5.1' # For 5.0.x, 5.1.x and 5.2.x
https://github.com/svenfuchs/rails-i18n